cmd/docker-machine-driver-kvm/Makefile

```diff
@@ -1,4 +1,4 @@
 default: build
 
 build:
-	GOGC=off go build -i -o docker-machine-driver-kvm
+	GOGC=off go build -tags libvirt.1.2.14 -i -o docker-machine-driver-kvm
```

Comment:
I rebased my changes against your master, this is the error I get when I try to build the code inside of 14.04:

```
# github.com/dhiltgen/docker-machine-kvm
./kvm.go:596: cannot use uint(libvirt.DOMAIN_INTERFACE_ADDRESSES_SRC_LEASE) (type uint) as type libvirt.DomainInterfaceAddressesSource in argument to d.VM.ListAllInterfaceAddresses
```

Do you think it's acceptable?